pyspectral/rayleigh.py

```diff
@@ -200,15 +202,15 @@ def get_effective_wavelength(self, bandname):
         if isinstance(bandname, str):
             bandname = BANDNAMES.get(self.sensor, BANDNAMES['generic']).get(bandname, bandname)
         elif isinstance(bandname, (float, integer_types)):
-            if not(0.4 < bandname < 0.8):
+            if not (0.4 < bandname < 0.8):
                 raise BandFrequencyOutOfRange(
                     'Requested band frequency should be between 0.4 and 0.8 microns!')
             bandname = get_bandname_from_wavelength(self.sensor, bandname, rsr.rsr)
 
         wvl, resp = rsr.rsr[bandname][
-            'det-1']['wavelength'], rsr.rsr[bandname]['det-1']['response']
+                        'det-1']['wavelength'], rsr.rsr[bandname]['det-1']['response']
 
-        cwvl = get_central_wave(wvl, resp, weight=1. / wvl**4)
+        cwvl = get_central_wave(wvl, resp, weight=1. / wvl ** 4)
```

Comment:
What is happening here? My flake8/pep8 is fine with the `wvl**4` - isn't that how it supposed to be?
https://realpython.com/python-pep8/